code/iaas/ha/src/main/java/io/cattle/platform/ha/monitor/impl/PingInstancesMonitorImpl.java

```diff
@@ -344,17 +354,20 @@ protected ReportedInstances getInstances(Ping ping) {
     }
 
     protected AgentAndHost loadAgentAndHostData(ImmutablePair<Long, String> agentIdAndHostUuid) {
```

Comment:
@alena1108 It is a pair because of how the LoadingCache works. The pair is the key to its map:
https://github.com/rancherio/cattle/pull/466/files#diff-ba9537ff87cd70d9ee6498f7391d5ea1R88
and I must provide a method that will load a value based off of that key.